R/create_dummy_package2.R

```diff
@@ -0,0 +1,110 @@
+
+    pid_jpg2 <- arcticdatautils::publish_object(mn,
+                                                path = "dummy2.jpg",
+                                                format_id = "image/jpeg")
+
+    data_pids <- c(pid_csv1, pid_csv2, pid_jpg1, pid_jpg2)
+
+    #import EML
+    eml_path_original <- "/home/isteves/datamgmt/inst/dummy_meta_full.xml"
+    #TODO change to:
+    #file.path(system.file(package = "arcticdatautils"), "dummy_meta_full.xml")
+
+    eml <- EML::read_eml(eml_path_original)
+    eml_path <- tempfile(fileext = ".xml") #for saving later
+
+    #add data tables + other entities to eml
+    eml@dataset@title[[1]]@.Data <- title
+
+    attr <-  data.frame(
+
+        attributeName = c('Date', 'Location', 'Salinity', 'Temperature'),
+        attributeDefinition = c('Date sample was taken on', 'Location code representing location where sample was taken', 'Salinity of sample in PSU', 'Temperature of sample'),
+        measurementScale = c('dateTime', 'nominal','ratio', 'interval'),
+        domain = c('dateTimeDomain', 'enumeratedDomain','numericDomain', 'numericDomain'),
+        formatString = c('MM-DD-YYYY', NA,NA,NA),
+        definition = c(NA,NA, NA, NA),
+        unit = c(NA, NA, 'dimensionless', 'celsius'),
+        numberType = c(NA, NA, 'real', 'real'),
+        missingValueCode = c(NA, NA, NA, NA),
+        missingValueCodeExplanation = c(NA, NA, NA, NA),
+
+        stringsAsFactors = FALSE)
+
+    Location <- c(CASC = 'Cascade Lake', CHIK = 'Chikumunik Lake', HEAR = 'Heart Lake', NISH = 'Nishlik Lake' )
+
+    fact <- data.frame(attributeName = 'Location', code = names(Location), definition = unname(Location))
+
+    dt1 <- arcticdatautils::pid_to_eml_datatable(mn,
+                                                 attributes = attr,
+                                                 factors = fact,
+                                                 pid = pid_csv1)
+    dt2 <- arcticdatautils::pid_to_eml_datatable(mn,
+                                                 attributes = attr,
+                                                 factors = fact,
+                                                 pid = pid_csv2)
+    eml@dataset@dataTable <- c(dt1, dt2)
+
+    oe <- arcticdatautils::pid_to_eml_other_entity(mn,
+                                                   pids = c(pid_jpg1,
+                                                            pid_jpg2))
+    eml@dataset@otherEntity@.Data <- oe
+
+    EML::write_eml(eml, eml_path)
+
+    pid_eml <- arcticdatautils::publish_object(mn,
+                                               path = eml_path,
+                                               format_id = "eml://ecoinformatics.org/eml-2.1.1")
+
+    #resource map together
+    resource_map_pid <- arcticdatautils::create_resource_map(
+        mn,
+        metadata_pid = pid_eml,
+        data_pids = data_pids)
+
+    file.remove(c("dummy1.csv", "dummy2.csv", "dummy1.jpg", "dummy2.jpg",
```

Comment:
`file.create` doesn't make the files in your global environment right?  I think you don't need this call

Reply:
checked with emily about this, and she said it's better practice to always clean up otherwise it sits in the memory for a while